### ISSUE TITLE
Incorrect messages about conflicting local server launchs

### ISSUE THREAD

**Issue by chanseokoh** [GoogleCloudPlatform/google-cloud-eclipse#1772]
- Steps to reproduce:
1. Create two HelloWorld projects, `Alice` and `Bob`.
2. Select `Alice`. _Run As_ > _App Engine_.
3. Select `Bob`. _Run As_ > _Run on Server_. Select _Manually define a new server_. Change _Server Port_ to, e.g., 8089. Click _Finish_.

![problem occurred _001](https://cloud.githubusercontent.com/assets/10523105/24872711/54ba3b5e-1dec-11e7-8b88-ad0f85a2f7d6.png)

The message mentions about the admin port, but it's basically irrelevant with dev appserver1. The port 8000 is also out of place. It's not true either that the storage path is conflicting, because AFAIK, for dev appsever1, the storage path will be inside `WEB-INF` of the deploy directory. In other words, there is no conflicting resources for dev appserver1, and we should be able to run `Alice` and `Bob` simultaneously. Is this correct @patflynn?

Note that, however, this may not necessarily be a regression in the sense that, previously with dev appserver2, we were not able to run `Alice` and `Bob` anyway. (There was no way for users to change the storage path in our Eclipse plugin.)

**Comment by elharo** [comment]
The issue is `LocalAppEngineServerLaunchConfigurationDelegate.checkConflicts()`. I wonder if that is overly aggressive in checking if two devappserver1's conflict.

**Comment by briandealwis** [comment]
We should push this conflict-checking into the devappserver-N-specific conflict checkers.

**Comment by elharo** [comment]
Makes sense. Can you elaborate? Is this something that should go into appengine-plugins-core or remain here?

**Comment by briandealwis** [comment]
I'm unsure if it's helpful for appengine-plugins-core.  We can only check against our own managed instances, so it wouldn't be useful for the maven or gradle plugins.  From what I recall of our conversations around the devappserver implementation, IntelliJ seemed to do launches quite a bit differently so I'm not sure how useful it would be there. @patflynn ?
